requirements.txt

```diff
@@ -15,7 +15,7 @@ google-api-python-client==2.116.0
 ibm_cloud_sdk_core==3.18.0
 ibm_vpc==0.20.0
 jinja2==3.1.4
-krkn-lib==2.1.7
+krkn-lib@git+https://github.com/krkn-chaos/krkn-lib.git@sippy_integration
```

Comment:
should we go ahead and create a krkn-lib tag for this branch since it got merged along with the version changes

Reply:
+1

Reply:
already done 2.1.8